t/features/builtins/operators.t

```diff
@@ -429,7 +445,11 @@ use _007::Test;
           (stexpr (postfix:() (identifier "say") (argumentlist (infix:+ (int 38) (str "4"))))))
         .
 
-    is-error $ast, X::TypeCheck, "adding non-ints is an error";
+    is-error
+        $ast,
+        X::TypeCheck,
+        "Type check failed in +; expected Val::Int but got Val::Str (Val::Str.new(value =>...)",
```

Comment:
This looks odd and not expected, but I think I'm going to let it through for now, because the real fix would be #245.